2024/incoming/annot-trees.pod

```diff
@@ -0,0 +1,167 @@
+Author: E. Choroba
+Title: Annotating Christmas Trees
+Topic: Natural Language Processing, Graph Visualisation
+
+=encoding utf8
+
+=head2 Lesser-known businesses
+
+Everybody knows that the North Pole™ produces and delivers Christmas gifts,
+but it’s not the sole business they’re into. In fact, they’ve recently engaged
+in production of Christmas ornaments and Christmas cards, too.
+
+As is common, starting a new business brings new problems. The elves needed a
+way to design decorated Christmas trees to evaluate their ornament proposals
+and their combinations. They started by planting small trees and decorating
+them by hand, but quickly found out this approach didn’t scale.
+
+At a C-level meeting, Santa listened to laments of the managing elf
+responsible for the ornaments and narrowed his eyes to another young elf at the other corner of the table.
+
+“You had something for arranging trees, right?” asked Santa.
+
+“Yes, but…” spluttered the elf.
+
+“Let’s meet after the lunch and see how we can share the knowledge,” commanded
+Santa.
+
+=head2 There’s trees and there’s trees
+
+In the afternoon (by the way, it was already dark, since it was the North Pole
+and summer was over) Santa met with the COO and CCO (where the second letter
+stands for Ornaments or Cards, respectively).
+
+“Can you show us what your department uses to visualise trees?” asked Santa,
```

Comment:
Turning to British spelling, are we?

Reply:
I think I keep it through the whole article 🇬🇧